lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -7,38 +7,43 @@ module FlashHelper
     #   <a href="" class="close">&times;</a>
     # </div>
     DEFAULT_KEY_MATCHING = {
-      :alert     => :alert,
-      :notice    => :success,
-      :info      => :info,
-      :secondary => :secondary,
-      :success   => :success,
-      :error     => :alert,
-      :warning   => :warning
+      :alert     => 'alert',
+      :notice    => 'success',
+      :info      => 'info',
+      :secondary => 'secondary',
+      :success   => 'success',
+      :error     => 'alert',
+      :warning   => 'warning'
     }
-    def display_flash_messages(key_matching = {})
+
+    def display_flash_messages(opts={})
```

Comment:
Good point.  Use `display_flash_messages(key_matching = {}, opts = {})`.  The options can be combined in version 2.0.